guides/common/modules/con_supported-client-architectures-for-configuration-management.adoc

```diff
@@ -0,0 +1,14 @@
+[id="Supported-Client-Architectures-for-Configuration-Management_{context}"]
+= Supported client architectures for configuration management
+
+You can use the following combinations of major versions of {RHEL} and hardware architectures for configuration management with {Project}.
+
+.Puppet Agent Support
```

Comment:
this heading does not match the file heading and introduction.

Reply:
Applied.